src/frontend/src/lib/constants/app.constants.ts

```diff
@@ -59,3 +62,5 @@ export const NANO_SECONDS_IN_MINUTE = NANO_SECONDS_IN_MILLISECOND * 1_000n * 60n
 // For some use case we want to display some amount to a maximal number of decimals which is not related to the number of decimals of the selected token.
 // Just a value that looks good visually.
 export const EIGHT_DECIMALS = 8;
+
+export const POUH_ENABLED = JSON.parse(import.meta.env.VITE_POUH_ENABLED ?? false) === true;
```

Comment:
Not for this PR but, maybe we can have `POUH_ENABLE` and `POUH_CREDENTIAL_TYPE` in the same module? Don't know if that makes sense. Again not this PR just reading one last time and asked my self.